app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -150,14 +152,12 @@ function DocumentCapture({ onStepChange = () => {} }: DocumentCaptureProps) {
         ) as FormStep[]);
   const reviewAfterFailedSteps = [reviewFormStep] as FormStep[];
   const reviewWithInPersonSteps = reviewAfterFailedSteps.concat(inPersonSteps);
-  const afterSubmissionErrorSteps = docAuthSeparatePagesEnabled
-    ? reviewAfterFailedSteps
-    : reviewWithInPersonSteps;
+  const afterSubmissionErrorSteps =
+    docAuthSeparatePagesEnabled && isSelfieCaptureEnabled && !isInPersonStepEnabled
+      ? reviewAfterFailedSteps
+      : reviewWithInPersonSteps;
```

Comment:
i was able to simplify lines 151-161 with the code snippet below to more easily follow what is happening here.  can you please update these lines to the following:

```
  let steps = documentsFormSteps;
  if (isInPersonStepEnabled){
    steps = inPersonSteps;
  } else if (submissionError) {
    steps = ([reviewFormStep] as FormStep[])
    if (!docAuthSeparatePagesEnabled || !isSelfieCaptureEnabled) { // figure out why this is here? .. doesn't seem right
      steps = steps.concat(inPersonSteps); 
    }
  }
```

it seems odd that that inPersonSteps is added to the steps when not doing split doc auth ... i suspect something is wrong here ... can we find out why?